### PR TITLE
Milestone 1 - Project Setup

### PR DESCRIPTION
**Pull request by gdumani**
This project is about the development of a simple TicTacToe game using Ruby.
In this Milestone the following objectives were achieved:

- Lib and bin directories were created
- main.rb file was added to the bin directory to start the project there.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "Hello World"
```
Comment (rammazzoti2000) [inline bin/main.rb]: On line 1 it should be `#!/usr/bin/env ruby` for the executable main file, followed on line 2 by `puts "Hello World"`

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello team 🙌, I am Alex your code reviewer for this project
Great job team this far, congrats on the `Readme` file, even though it was not yet requested a readme file, yours is already great 💪 , also the PR summary is descriptive and overall the repo looks professional.
There is just on small thing to adjust before approving this project, checkout the comments bellow 👇 
## 🛑 Required Changes 🚧 👷 
- [x] The `GitHub Actions` configuration is not complete, you are missing the first step, please check again [the official repo](https://github.com/microverseinc/linters-config/tree/master/ruby) and make sure you have your [`.github/workflows/linters.yml`](https://github.com/microverseinc/linters-config/blob/master/ruby/.github/workflows/linters.yml) file configured as well
image
- [x] Also the first line in the executable main file must be `#!/usr/bin/env ruby` followed by the `
puts "Hello world"` statement in the second line
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of **_alexandru_** if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Comment by epinczinger** [comment]
- Added linters.yml and fixed rubocop errors.
-First-line on main.rb has been added.

**Comment by lekegitrepo** [comment]
### Good job :+1: Team.
#### This project is approved, go ahead and merge the branch now.

#### _Reviewed by Temitayo Adeleke_.

**Review by lekegitrepo — APPROVED** [review]
#### Status: Approve

**Pull request merged**